src/win/tty.c

```diff
@@ -852,6 +925,17 @@ void uv_process_tty_read_raw_req(uv_loop_t* loop, uv_tty_t* handle,
         const char* vt100;
         size_t prefix_len, vt100_len;
 
+        /* The key codes for the '/' and '_' keys depend on the user's keyboard
+         * layout.  Since the layout may change, and is actually very likely to
+         * change for devs that don't use EN-US by default, just look it up each
+         * loop and replace them with the ASCII codes, which get_vt100_fn_key
+         * knows how to translate. */
+        if (KEV.wVirtualKeyCode == LOBYTE(VkKeyScanA('/'))) {
```

Comment:
I'm not familiar with this system (or non EN-US keyboards). But why isn't it reporting `VK_DIVIDE`?

Reply:
On my keyboard, `VK_DIVIDE` is the numpad `/` key, but the one next to shift shows up as `VK_OEM_2` which MSDN says can represent different keys depending on layout. I forgot to handle `VK_DIVIDE` here, at least it's not working on my copy of Neovim with these changes, so I'll need to fix that.

Also see [here](https://github.com/microsoft/terminal/blob/322989d01701cc688593f4b3f06e535b77a8810f/src/terminal/input/terminalInput.cpp#L329) where they do the same in the official Windows terminal.

Reply:
Actually it looks like I need to dig into this a little more, it's resolving to the wrong key on the Spanish keyboard layout.